lib/new_relic/agent/instrumentation/active_job.rb

```diff
@@ -58,14 +58,21 @@ def self.perform(job, block)
         end
 
         def self.run_in_trace(job, block, event)
-          trace_execution_scoped("MessageBroker/#{adapter}/Queue/#{event}/Named/#{job.queue_name}") do
+          trace_execution_scoped("MessageBroker/#{adapter}/Queue/#{event}/Named/#{job.queue_name}",
```

Comment:
I'm puzzled by this metric's usage of the MessageBroker format. According to the [agent specs (internal)](https://source.datanerd.us/agents/agent-specs/blob/0c52d16310bcef40cea84873408389f3d1597a98/APIs/messaging.md), ActiveJob does not seem like a MessageBroker.

I think we should look into this more. I've made a technical debt ticket to triage.
https://github.com/newrelic/newrelic-ruby-agent/issues/1151

Reply:
Thank you for filing that one. I could see how someone may have looked at a Redis backed queueing system for ActiveJob and saw a resemblance to an existing MQ type system and stretched "message broker" to cover the use case. Both systems feature a regulation of distributed clients operating over multiple queues. But there's certainly not (in the default case) any actual message broking going on.